com.unity.render-pipelines.universal/Editor/UniversalRendererDataEditor.cs

```diff
@@ -74,6 +81,13 @@ public override void OnInspectorGUI()
             }
             EditorGUI.indentLevel--;
             EditorGUILayout.Space();
+#if ENABLE_RENDER_PASS_UI
```

Comment:
At this time this feature will regress the performance on these platforms, so we shouldn't really show it. This could be done after lazy merging is done, but for now - lets keep it hidden